packages/gguf/src/gguf.ts

```diff
@@ -205,38 +227,41 @@ function readMetadataValue(
 }
 
 export async function gguf(
-	url: string,
+	uri: string,
 	params: {
 		/**
 		 * Custom fetch function to use instead of the default one, for example to use a proxy or edit headers.
 		 */
 		fetch?: typeof fetch;
 		additionalFetchHeaders?: Record<string, string>;
 		computeParametersCount: true;
+		localFile?: boolean;
 	}
 ): Promise<GGUFParseOutput & { parameterCount: number }>;
 export async function gguf(
-	url: string,
+	uri: string,
 	params?: {
 		/**
 		 * Custom fetch function to use instead of the default one, for example to use a proxy or edit headers.
 		 */
 		fetch?: typeof fetch;
 		additionalFetchHeaders?: Record<string, string>;
+		localFile?: boolean;
 	}
 ): Promise<GGUFParseOutput>;
 export async function gguf(
-	url: string,
+	uri: string,
 	params?: {
 		/**
 		 * Custom fetch function to use instead of the default one, for example to use a proxy or edit headers.
 		 */
 		fetch?: typeof fetch;
 		additionalFetchHeaders?: Record<string, string>;
 		computeParametersCount?: boolean;
+		localFile?: boolean;
```

Comment:
```suggestion
		/** By default this function will not look at the local file system for security reasons */
		allowLocalFile?: boolean;
```

Also, when detecting a relative url in a backend context, we should probably throw an explicit error if this is not set ("Set allowLocalFile to ..."). So that the user can know why it failed, since the user may not know about the option

Reply:
I implemented it in https://github.com/huggingface/huggingface.js/pull/656/commits/a0e174e581f9572da8ebc15e9c9c32c34ed05b24 , please have a look